app/src/main/java/org/secuso/privacyfriendlytodolist/view/dialog/ProcessTodoTaskDialog.java

```diff
@@ -316,7 +316,7 @@ public boolean onMenuItemSelected(int featureId, MenuItem item) {
                 this.selectedListID = tl.getId();
                 listSelector.setText(tl.getName());
             } else if (item.getTitle() == getContext().getString(R.string.to_choose_list)){
-                this.selectedListID = -3;
+                this.selectedListID = 1;
```

Comment:
you can't assume the default-list will have the id 1.

Reply:
We assumed that the default-list will have the id 1 because it will be always the first list when the user starts using the app

Reply:
Yes, but people are already using the app. So either you have to change the ids of the existing lists or you cant assume that the default list will have the id 1. Also you have to consider the case, that there are lists already and you still have to add the default list. People should not be forced to completely wipe their tasks to update the app.